examples/hooks/components/InfiniteHits.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { Hit as AlgoliaHit } from '@algolia/client-search';
+import {
+  useInfiniteHits,
+  UseInfiniteHitsProps,
+} from 'react-instantsearch-hooks';
+
+import { cx } from '../cx';
+
+export type InfiniteHitsProps<THit> = React.ComponentProps<'div'> &
+  UseInfiniteHitsProps & {
+    hitComponent: (props: { hit: THit }) => JSX.Element;
+  };
+
+export function InfiniteHits<THit extends AlgoliaHit<Record<string, unknown>>>({
+  hitComponent: Hit,
```

Comment:
I would keep the name `hitComponent` or just rename it to `Hit` component, otherwise, at least for me, it's too confusing to remember if you're dealing with the type, or the prop.

Reply:
You meant `HitComponent` right?